index.js

```diff
@@ -113,9 +114,12 @@ client.on('groupStatusChange', (guild, group, enabled) => {
 	logger.info(consoleMessage);
 });
 
-client.login(config.discord.botToken);
+client.login(config.discord.botToken).then((token) => {
+	store.guild = client.guilds.get(config.discord.channelId);
```

Comment:
Since `guilds` is of type [collection](https://discord.js.org/#/docs/main/stable/class/Collection) the `.get()` will work on the [snowflake](https://discord.js.org/#/docs/main/stable/typedef/Snowflake) of the collection. This means that `.get(config.discord.channelId)` will not work since the `config.discord.channelId` is the id of the main chat for Lucy and not the snowflake of the collection.
You will have to iterate over the entries of `guilds` and either find the guild that has the channel matching the channelId or add a `guildId to the config with the id of the guild and retrieve the guild by that.